app/views/tag/_location.html.erb

```diff
@@ -4,7 +4,11 @@
 <script>
   jQuery(document).ready(function() {
     $('.blurred-location-input').click(function createBlurredLocationInput() {
-      $.ajax('/locations/modal')
+      <%if parent==:profile%>
```

Comment:
I think this could become `<% if params[:action] == "profile" %>` so as to ensure we're in this action:

https://github.com/publiclab/plots2/blob/aa1687149828a17301c8f9d875c22502e5ca4b43/app/controllers/users_controller.rb#L103

Does that make sense? Also, sorry to be picky but if you could add spaces inside the `<% end %>` and the `if` it's a bit more readable. Much appreciated!!!